pkg/chains/storage/docdb/docdb.go

```diff
@@ -262,6 +266,18 @@ func populateMongoServerURL(ctx context.Context, cfg config.Config) error {
 	logger := logging.FromContext(ctx)
 	mongoEnv := "MONGO_SERVER_URL"
 
+	if cfg.Storage.DocDB.MongoServerURLPath != "" {
+		logger.Infof("setting %s from storage.docdb.mongo-server-url-path: %s", mongoEnv, cfg.Storage.DocDB.MongoServerURLPath)
+		mongoServerURL, err := getMongoServerURLFromPath(cfg.Storage.DocDB.MongoServerURLPath)
+		if err != nil {
+			return err
+		}
+		if err := os.Setenv(mongoEnv, mongoServerURL); err != nil {
+			return err
+		}
+		return nil
+	}
+
 	if cfg.Storage.DocDB.MongoServerURLDir != "" {
 		logger.Infof("setting %s from storage.docdb.mongo-server-url-dir: %s", mongoEnv, cfg.Storage.DocDB.MongoServerURLDir)
```

Comment:
It would be good to use constant `docDBMongoServerURLDirKey` instead of key directly

Reply:
Same here